nvflare/private/fed/client/client_app_runner.py

```diff
@@ -103,16 +104,11 @@ def create_run_manager(self, args, conf, federated_client, workspace):
             conf=conf,
         )
 
-    def start_command_agent(self, args, client_runner, federated_client, fl_ctx):
-        while federated_client.cell is None:
-            self.logger.info("Waiting for the client cell to be created ....")
-            time.sleep(1.0)
-
+    def start_command_agent(self, args, federated_client, fl_ctx):
         # Start the command agent
-        # self.command_agent = CommandAgent(federated_client, int(args.listen_port), client_runner)
-        self.command_agent = CommandAgent(federated_client, client_runner)
+        self.command_agent = CommandAgent(federated_client)
         self.command_agent.start(fl_ctx)
-        client_runner.command_agent = self.command_agent
+        self.command_agent.register_cell_cb()
```

Comment:
Why do you register_cell_cb in here? It is already called in command_agent.start().

Reply:
Removed the duplicate calls.